slatedb/src/config.rs

```diff
@@ -243,6 +243,18 @@ pub struct ReadOptions {
     /// Whether to include dirty data in the scan. "dirty" means that the data is not considered
     /// as "committed" yet, whose seq number is greater than the last committed seq number.
     pub dirty: bool,
+    /// Whether or not fetched blocks should be cached
+    pub cache_blocks: bool,
```

Comment:
nit: how about calling this parameter direct? This might be similar to the semantics of `O_DIRECT` in Linux syscalls.

(cache_blocks is a nice name imo, i just remembered this naming convention from Linux.

Reply:
I'm +1 to leaving it as `cache_blocks` or consistency

Reply:
oh, thank you for the background info! i didn't know about `cache_blocks` in ScanOptions before.

+1 for consistency 😆

Reply:
I don't love `direct` since our cache isn't really a write-back cache like the page cache. SlateDB writes are always "direct" in the sense that they always go to storage. `cache_blocks` seems more precise to me.